### PR TITLE
docs: runnable example of usage

### PR DESCRIPTION
**Pull request by jirikuncar**
Signed-off-by: Jiri Kuncar jiri.kuncar@cern.ch

### CODE REVIEW

File: invenio_documents/__init__.py [hunk 1]
```diff
@@ -22,7 +22,122 @@
...
+
+>>> from invenio_records.api import Record
+>>> record = Record.create(
+... {'title': 'Test', 'files': [{'uri': '/tmp/hello.txt'}]})
+
+
+Creating Documents
+~~~~~~~~~~~~~~~~~~
+
+First we need a pointer to URI stored in metadata.
```
Comment (lnielsen) [inline invenio_documents/__init__.py]: First we need a pointer to `the` URI stored in `the` metadata.

File: invenio_documents/__init__.py [hunk 2]
```diff
@@ -22,7 +22,122 @@
...
+
+>>> uri_pointer = '/files/0/uri'
+
+With given ``uri_pointer`` we can instantiate ``Document`` class
```
Comment (lnielsen) [inline invenio_documents/__init__.py]: With given _the_  `uri_pointer` we can instantiate _a_ `Document` class
and write something to it.

**Comment by lnielsen** [comment]
LGTM besides tests.

**Comment by jirikuncar** [comment]
@lnielsen fixed. Thanks :rocket:

**Pull request merged**